mlflow/models/cli.py

```diff
@@ -30,7 +30,6 @@ def commands():
 @cli_args.HOST
 @cli_args.TIMEOUT
 @cli_args.WORKERS
-@cli_args.NO_CONDA
 @cli_args.ENV_MANAGER
```

Comment:
shall we change the default value of `ENV_MANAGER` option to be "virtualenv" ?

Reply:
We did in https://github.com/mlflow/mlflow/pull/6501.